test/parallel/test-stream-pipe-unpipe-streams.js

```diff
@@ -31,3 +31,57 @@ source.unpipe(dest2);
 source.unpipe(dest1);
 
 assert.strictEqual(source._readableState.pipes, null);
+
+{
+  // test `cleanup()` if we unpipe all streams.
+  const source = Readable({ read: () => {} });
+  const dest1 = Writable({ write: () => {} });
+  const dest2 = Writable({ write: () => {} });
+
+  let destCount = 0;
+  const srcCheckEventNames = ['end', 'data'];
+  const destCheckEventNames = ['close', 'finish', 'drain', 'error', 'unpipe'];
+
+  const checkSrcCleanup = common.mustCall(() => {
+    assert.strictEqual(source._readableState.pipes, null);
+    assert.strictEqual(source._readableState.pipesCount, 0);
+    assert.strictEqual(source._readableState.flowing, false);
+
+    srcCheckEventNames.forEach((eventName) => {
+      assert.strictEqual(
+        source.listenerCount(eventName), 0,
+        `source's '${eventName}' event listeners don't be cleaned up`
```

Comment:
Nit: replace "don't be cleaned up" with "not removed", same below.

Reply:
ok~